interfaces/Portalicious/src/app/pages/project-payment/components/single-payment-export/single-payment-export.component.ts

```diff
@@ -35,6 +35,7 @@ import { ToastService } from '~/services/toast.service';
 export class SinglePaymentExportComponent {
   projectId = input.required<string>();
   paymentId = input.required<string>();
+  hasExportFileIntegration = input<boolean | undefined>(false);
```

Comment:
```suggestion
  hasExportFileIntegration = input<boolean>(false);
```

I think? Then `false` is the default when it is undefined.

Reply:
Yeah I tried that but I get the following error then:
![image](https://github.com/user-attachments/assets/f56c3070-baf8-422c-a079-d473d9d15a8a)

Reply:
Because `projectHasFspWithExportFileIntegration` returns `undefined` but it should return a `boolean`